src/SINDYPI_fit.jl

```diff
@@ -0,0 +1,457 @@
+##FIT FUNCTION
+
+function fit!(X::AbstractArray, Y::AbstractArray, maxiter, lambda, Polyorder, Trigorder)
+    using LinearAlgebra

```

Comment:
If this would be a regression, you would not need the using here ( already in Optimise.jl ).